android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.kt

```diff
@@ -343,7 +343,7 @@ class Kernel : KernelInterface() {
 
   val reactRootView: ReactRootView
     get() {
-      val reactRootView: ReactRootView = ReactUnthemedRootView(context)
+      val reactRootView: ReactRootView = ReactUnthemedRootView(activityContext)
```

Comment:
I think @Kudo  mentioned that he was unsure how this might affect other functionality on Android - we'll have to note to be extra diligent during QA for the next SDK release